tests/checks/tmux-empty-prompt.fish

```diff
@@ -0,0 +1,77 @@
+#RUN: %fish %s
+#REQUIRES: command -v tmux
+
+isolated-tmux-start -C '
+    function fish_prompt; end
+    function fish_right_prompt
+        set -q right_prompt
+        or echo right-prompt
```

Comment:
it's weird that presence of `$right_prompt` turns *off* right prompt.
I'd probably expect it other way round.
But actually, do we need to overload the ctrl-g binding like this?

Maybe create separate bindings, one for turning on right prompt
and another for turning it off;
that way the test should be easier to read?